### DISCUSSION TITLE
Outline onion skin

### DISCUSSION THREAD

**Discussion by Storm840** [keiwando/toonsquid#606] [Ideas]
Hi, i would really like an outline option for the onion skin like in "wick editor"
![image](https://github.com/keiwando/toonsquid/assets/142567495/4a339da5-a9e4-4c93-a421-ba673a4b4a26)
![image](https://github.com/keiwando/toonsquid/assets/142567495/0660983c-3e18-4780-b96d-bdc7c6fa0e32)

**Comment by DWrightAnimation** [comment]
Honestly, this would be useful for when someone wants to add an inbetween, but have already colored the previous frame.
